noxfile.py

```diff
@@ -10,20 +10,22 @@
 
 @nox.session(python=["3.8", "3.9", "3.10", "3.11", "pypy3.8", "pypy3.9"])
 def tests(session: nox.Session) -> None:
```

Comment:
I don't like that the commands are different than py-template, but this is what we have to do to get coverage report for pytest-plugins. We do same for `pytest-test-utils`: https://github.com/iterative/pytest-test-utils/blob/64e8889564dc7b2c005d02464a7feec07cc9c602/noxfile.py#L18.